docs/content/en/docs/Developers/factorycube-edge/cameraconnect/index.md

```diff
@@ -204,4 +277,5 @@ The input is not case sensitive. Please follow the example format below.
 
 ## Credits
 
-Based on the Bachelor Thesis from [Patrick Kunz](../../../publications/plug-and-play-visual-quality-inspection).
+Based on the Bachelor Thesis from 
+[Patrick Kunz](/docs/content/en/docs/publications/plug-and-play-visual-quality-inspection).
```

Comment:
please do not adjust it like this. This will not work in hugo. use `/docs/publications/plug-and-play-visual-quality-inspection ` instead

Reply:
changed link to
/docs/publications/plug-and-play-visual-quality-inspection